test/integration/02e-onchain-send.spec.ts

```diff
@@ -62,6 +62,7 @@ afterAll(async () => {
 
 const amount = 10040 // sats
 
+// Starts to fail from here...
```

Comment:
if it is a timeout fail is because waitUntilBlockHeight. Basically in lnd the "pruning channel graph" is taking more than the timeout in jest.
- you can check docker logs to verify that this is the case
- increase timeout in jest setup https://github.com/GaloyMoney/galoy/blob/main/test/jest.setup.js

Reply:
Thanks for this, modifying the timeout has allowed me to retry the tests when these show the following error:

```
FAIL test/integration/02a-fund-bitcoind.spec.ts (36.247 s)
  ✓ funds bitcoind wallet (3697 ms)
  ✕ funds outside lnd node (30268 ms)

  ● funds outside lnd node

    thrown: "Exceeded timeout of 30000 ms for a test.
    Use jest.setTimeout(newTimeout) to increase the timeout value, if this is a long-running test."

      66 | })
      67 |
    > 68 | it("funds outside lnd node", async () => {
```

But this seems to be "unrelated?" to the the main failure that the tests are showing, which is that the balance sheet is no longer balanced.

Reply:
@jotapea FYI tests are NOT idempotent so you always have to run `make reset-deps` between runs.

Reply:
Yes I'm running them with 'make integration' the first time and then 'make reset-integration' the rest of the times.